doc/man3/HMAC.pod

```diff
@@ -142,11 +143,11 @@ RFC 2104
 
 =head1 SEE ALSO
 
-L<SHA1(3)>, L<evp(7)>
+L<SHA1(3)>, EVP_Q_mac(), L<evp(7)>
```

Comment:
(3)

Reply:
Fixed.
Interesting that `make doc-nits` does not report this.